File: DefaultBrowser/DefaultBrowser/DefaultBrowser.py
import win32api


def ShellExecute_url(url):
    win32api.ShellExecute(0, 'open', url, '', '', 1)


if __name__ == '__main__':
    ShellExecute_url('https://time.navyism.com/?host=http://naver.com')
    pass

File: DefaultBrowser/DefaultBrowser.go
// DefaultBrowser
package DaeseongLib

import (
	"fmt"
	"os/exec"
)

func Webbrowser(sUrl string) (err error) {
	_, err = exec.Command("cmd", "/c", "start", sUrl).Output()
	return err
}

func main() {
	err := Webbrowser("https://time.navyism.com/?host=http://naver.com")
	if err != nil {
		fmt.Println(err)
	}
}
